doc/devel/documenting_mpl.rst

```diff
@@ -213,38 +220,52 @@ is better than:
 In addition, since underscores are widely used by Sphinx itself, use
 hyphens to separate words.
 
+.. _referring-to-other-code:
 
 Referring to other code
 -----------------------
 
 To link to other methods, classes, or modules in Matplotlib you can use
 back ticks, for example:
 
-.. code-block:: python
+.. code-block:: rst
 
-  `~matplotlib.collections.LineCollection`
+  `matplotlib.collections.LineCollection`
 
-returns a link to the documentation of
-`~matplotlib.collections.LineCollection`.  For the full path of the
-class to be shown, omit the tilde:
+generates a link like this: `matplotlib.collections.LineCollection`.
 
-.. code-block:: python
+*Note:* We use the sphinx setting `default_role = 'obj'` so that you don't
```

Comment:
why not make these `.. note::` so it'll render that way in the site?

Reply:
This is rather a side note. It's not something you have to know to be able to write documentation. As such a `.. note::` admonition would give it too much attention.